tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -357,7 +358,7 @@ public void testCGOImportInNonTestFile() {
     myFixture.configureByText("a.go", "package a; import \"C\"");
     myFixture.checkHighlighting();
   }
-  
```

Comment:
I know but that's strange it happens. Shouldn't the code style be applied from the plugin repository? I haven't done anything special for this and I have the other settings such as always trim whitespaces and so on applied because of the other projects I'm working on.
Would it be ok if I run formatting on the whole project once when you guys aren't developing anything on it so I don't interfere?